src/formatDate.js

```diff
@@ -8,7 +8,42 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const toSeparator = toFormat.pop();
```

Comment:
The code assumes that the last element of `fromFormat` and `toFormat` is a separator. Ensure that this assumption is valid or add error handling to manage cases where the separator is not provided.